src/components/table/style-utilities.jsx

```diff
@@ -0,0 +1,132 @@
+import c from 'color';
```

Comment:
There are a lot of classes left in this file that does not utilize jss (`xs`/`sm`/..., `mono`, `ellipsis`, etc). Is that necessary to make the table styling work?